service/templates/projects.html

```diff
@@ -140,6 +140,16 @@ <h2> Settings for [[displayName]]</h2>
         </div>
       </div>
 
+      <div class="form-group">
+        <div class="col-lg-2 col-header">
+          <label class="control-label">Spawn Host Script Path</label>
+          <div class="muted small">This is the setup script to optionally run on spawn hosts created from tasks.</div>
```

Comment:
Since the script isn't distro-specific, do we need to make sure that it's always bash? Otherwise I think there will be confusion as to what shell it gets run in

Reply:
Oh hmm, that's an interesting question. @Kimchelly might have an opinion on this? [The current job ](https://github.com/evergreen-ci/evergreen/blob/af0b5c1505f6a175316224de9c5cf2f8c694b738/units/host_execute.go#L67)that runs the script does look at the host distro to decide how to run the script. Would a script work for some distros and not others?

Reply:
The distros are currently configured to all run shell scripts in bash, but it's not really a guarantee that it'll be true forever.

Reply:
I'll add "bash" to the description and that should be sufficient for now.